Installation fails on OSX with "signature not found in transparency log"

```
Step 1: Downloading: chainloop-cli-0.9.0-darwin-arm64.tar.gz
Done...

Step 1.2: Verifying checksum
Checksum OK

Step 1.3: Verifying signature
Error: signature not found in transparency log
main.go:74: error during command execution: signature not found in transparency log

```

Cosign version:
```
GitVersion:    2.0.2
GitCommit:     871448050b924a7946ebe47678f23aae09ef432d
GitTreeState:  "clean"
BuildDate:     2023-04-24T17:31:42Z
GoVersion:     go1.20.3
Compiler:      gc
Platform:      darwin/arm64
```

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -19,20 +19,19 @@ jobs:
       packages: write # to push container images
       pull-requests: write
     env:
-      CHAINLOOP_VERSION: 0.9.0
+      CHAINLOOP_VERSION: 0.9.1
       CHAINLOOP_ROBOT_ACCOUNT: ${{ secrets.CHAINLOOP_ROBOT_ACCOUNT }}
       CONTAINER_IMAGE_CP: ghcr.io/chainloop-dev/chainloop/control-plane:${{ github.ref_name }}
       CONTAINER_IMAGE_CAS: ghcr.io/chainloop-dev/chainloop/artifact-cas:${{ github.ref_name }}
     steps:
-      - name: Install Chainloop
-        run: |
-          curl -sfL https://docs.chainloop.dev/install.sh | bash -s -- --version v${{ env.CHAINLOOP_VERSION }}
-
-      # TODO(miguel) Move once the next release of chainloop is out and recorded in rekor
       - name: Install Cosign
         uses: sigstore/cosign-installer@main
         with:
           cosign-release: 'v2.0.2'
+
+      - name: Install Chainloop
+        run: |
+          curl -sfL https://docs.chainloop.dev/install.sh | bash -s -- --version v${{ env.CHAINLOOP_VERSION }}
 
       - name: Download jq
         run: |

